contracts/script/GenerateAlloc.s.sol

```diff
@@ -332,6 +334,19 @@ contract GenerateAlloc is Script {
         console2.log("Create3 deployed at:", Predeploys.Create3);
     }
 
+    function setERC6551() internal {
+        address tmp = address(new ERC6551Registry());
```

Comment:
Have we verified that `ERC6551Registry` does not set storage values in its constructor?
If it does, we have to set the same values in the correct addresses, using `vm.store`

Reply:
Confirmed.  ERC6551Registry has no any storage.